src/docs/sdk/research/performance/index.mdx

```diff
@@ -66,7 +66,7 @@ The implementation of the actual `trace` function is relatively simple (see [a P
 
 The following two examples synthesize the scope propagation issues.
 
-### 1. Cannot Determine Current Span
+### Cannot Determine Current Span
```

Comment:
Removed the numbers from headings because there are too many similar `<h3>` headings under "Span Ingestion Model", and it would be silly and error prone to manually number all of those.

If order would ever change, it would break anchors/links.

Reply:
Moved to https://github.com/getsentry/develop/pull/387